src/Controller/EnMarche/UserController.php

```diff
@@ -39,12 +40,12 @@ public function profileOverviewAction(): Response
      * @Route("/mes-dons", name="app_user_profile_donation")
      * @Method("GET")
      */
-    public function profileDonationAction(DonationRepository $donationRepository): Response
+    public function profileDonationAction(DonationRepository $donationRepository, TransactionRepository $transactionRepository): Response
     {
         $userEmail = $this->getUser()->getEmailAddress();
 
         return $this->render('user/my_donation.html.twig', [
-            'donations' => $donationRepository->findByEmailAddressOrderedByDonatedAt($userEmail, 'DESC'),
+            'transactions_success' => $transactionRepository->findAllSuccessfulTransactionByEmail($userEmail),
```

Comment:
For more consistent finally I think we can rename that to `findAllSuccessTransactionByEmail`